test/run.js

```diff
@@ -165,11 +165,10 @@ enqueue(convert, [
   "./test/class.es5.js"
 ]);
 
-function convertWithSpread(es6File, es5File, callback) {
+function convertWithParamsTransform(es6File, es5File, callback) {
```

Comment:
I really appreciate your attention to detail, such as taking the time to update the test harness so it doesn't accidentally rely on the `@babel/plugin-transform-spread` transform.